.github/workflows/lint.yaml

```diff
@@ -27,6 +27,9 @@ jobs:
         uses: github/super-linter@v4
         env:
           VALIDATE_ALL_CODEBASE: false
-          DEFAULT_BRANCH: main
+          DEFAULT_BRANCH: master
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           VALIDATE_SQLFLUFF: false
+          VALIDATE_MARKDOWN: false
```

Comment:
The Markdown linter doesn't understand how to deal with the Jinja templating in dbt Markdown files, and as far as I can tell there's no way to exclude files from markdownlint via the superlinter config, so I went ahead and turned it off.